File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L4/01/L4.ts
import * as readline from 'readline';

function hitBmi(berat: number, tinggi: number): number {
    return berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254));
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter weight in pounds: ', (weightStr) => {
    const berat = parseFloat(weightStr);

    rl.question('Enter feet: ', (feetStr) => {
        const feet = parseFloat(feetStr);

        rl.question('Enter inches: ', (inchesStr) => {
            const inci = parseFloat(inchesStr);
            const tinggi = feet * 12 + inci;

            const bmi = hitBmi(berat, tinggi);
            console.log(`BMI is ${bmi}`);

            if (bmi < 18.5) {
                console.log('Underweight');
            } else if (bmi < 25) {
                console.log('Normal');
            } else if (bmi < 30) {
                console.log('Overweight');
            } else {
                console.log('Obese');
            }

            rl.close();
        });
    });
});

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-03/plagiarized/L4/01/L4.kt
package IR

import java.util.*

object L4 {
    fun hitBmi(berat: Double, tinggi: Double): Double {
        return berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254))
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        var berat = 0.0
        var inci = 0.0
        var tinggi = 0.0

        var feet = 0.0

        print("Enter weight in pounds: ")
        berat = sc.nextDouble()

        print("Enter feet: ")
        feet = sc.nextDouble()
        print("Enter inches: ")
        inci = sc.nextDouble()
        tinggi = feet * 12 + inci

        println("BMI is " + hitBmi(berat, tinggi))
        if (hitBmi(berat, tinggi) < 18.5) {
            println("Underweight")
        } else if (hitBmi(berat, tinggi) < 25) {
            println("Normal")
        } else if (hitBmi(berat, tinggi) < 30) {
            println("Overweight")
        } else {
            println("Obese")
        }
    }
}
